package.json

```diff
@@ -37,7 +37,11 @@
     "@auth0/nextjs-auth0": "^1.6.1",
     "@prisma/client": "latest",
     "dayjs": "^1.10.7",
+    "imagemin-mozjpeg": "^10.0.0",
+    "imagemin-optipng": "^8.0.0",
+    "imagemin-svgo": "^10.0.1",
     "next": "^12.0.7",
+    "next-optimized-images": "^2.6.2",
```

Comment:
I have used this plugin in the past – it’s been pretty helpful, especially before Next.js got [native image optimisation support](https://nextjs.org/docs/basic-features/image-optimization). Not sure if we need all this complexity now – I don’t expect us to have a lot of images that benefit from `next-optimized-images`. Downloading or building `imagemin-*` binaries may significantly slow down `yarn install` while giving us very little (if any?) benefit.

The SVG looks optimised to me already. We can split this PR into two, merge the one with an SVG and then delay the plugin as much as we can.